lib/hammer_cli_katello/content_view_version.rb

```diff
@@ -385,6 +385,145 @@ def export_json(content_view_version, repositories)
       end
     end
 
+    class ImportCommand < HammerCLIForeman::Command
+      include HammerCLIForemanTasks::Async
+      include HammerCLIKatello::LocalHelper
+      include HammerCLIKatello::ApipieHelper
+
+      attr_accessor :export_tar_dir, :export_tar_file, :export_tar_prefix
+
+      desc _('Import a content view version')
+
+      command_name "import"
+
+      success_message _("Content view imported.")
+      failure_message _("Could not import the content view.")
+
+      option "--organization-id", "ORGANIZATION_ID", _("Organization numeric identifier")
+      option(
+        '--export-tar',
+        'EXPORT_TAR',
+        _("Location of export tar on disk")
+      )
+
+      validate_options do
+        option(:option_export_tar).required
+        option(:option_organization_id).required
+      end
+
+      build_options
+
+      class OrganizationIdParamSource
+        def initialize(command)
+          @command = command
+        end
+
+        def get_options(_defined_options, result)
+          if result['option_organization_id'].nil? && result['option_organization_name']
+            result['option_organization_id'] = @command.resolver.organization_id(result)
+          end
+          result
+        end
+      end
+
+      def option_sources
+        super << OrganizationIdParamSource.new(self)
+      end
```

Comment:
You don't need to add to option_sources or define a custom option source, afaict. Did you find it necessary for some reason? In addition, you're doing no ID resolution, so I'm not sure why there's logic to support organization name on [L422](https://github.com/Katello/hammer-cli-katello/pull/592/files#diff-c16d36ded485c9e83859c586343e3bb6R422).

Reply:
Just following patterns I see in the the code in other places. What is the alternative?

Reply:
Remove it all. It works fine without it (try for yourself!).